src/doc/CONTRIBUTE.md

```diff
@@ -1,58 +1,72 @@
 # Contribute
 
+<!--
 This chapter should display all needed information to set up and run the project and how a new joiner can start contributing to the it. Please provide a short introduction here.
+-->
 
 ## Chapters
```

Comment:
you did not touched this part but I think while we are add it to improve the files we should change this to `content` like it is in the other MD files